docs/guides/run-preinstalled-wda.md

```diff
@@ -49,9 +49,9 @@ can install the WebDriverAgent package.
 
 Some tools let you set an arbitrary bundle identifier (`CFBundleIdentifier` for the `Info.plist`) and sign it with the bundle identifier.
 It may not have `.xctrunner` as the bundle identifier.
-Default `appium:updatedWDABundleId` takes care of the hidden automatic suffix, `.xctrunner`, by Xcode, but it could break the non- `.xctrunner` package name case.
-`appium:doNotAddXctrunnerSuffix` capability will prevent adding the suffix.
-A session launches preinstalled bundle id that does not include `.xctrunner` with `"appium:doNotAddXctrunnerSuffix": true`.
+XCUITest driver takes care of the hidden automatic suffix, `.xctrunner`, by Xcode by default with  `appium:updatedWDABundleIdSuffix`.
```

Comment:
XCUITest driver automatically adds the `.xctrunner` suffix to the provided bundle identifier unless a different suffix is specified by  the `appium:updatedWDABundleIdSuffix` capability. If the value is an empty string then no suffix will be added to the provided bundle identifier.